example/config/local.yaml

```diff
@@ -0,0 +1 @@
+name: local
```

Comment:
I suppose this property is only for test purposes, right?

Reply:
Yes, this is merely an example file.